_includes/use-cases/use-case-transport-create-own-relationshiptemplate.md

```diff
@@ -0,0 +1,22 @@
+Create a relationship template.
+
+A relationship template can be used by a third party to initiate a relationship with you.
+Read [more]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate).
```

Comment:
Parameters, Success and Failure headlines (and descriptions) are missing.

Reply:
This is done in the new branch.